.github/workflows/wordpress-version-checker.yml

```diff
@@ -18,6 +19,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: WordPress version checker
-        uses: skaut/wordpress-version-checker@v2.0.0
+        uses: skaut/wordpress-version-checker@v2.2.1
```

Comment:
any reason to just truncate that to `v2` and get all minor updates along the way?

Reply:
I initially tried that but there isn't a valid `v2` release to make that work. If you try to go to https://github.com/skaut/wordpress-version-checker/tree/v2/ it will 404, so the only thing I could get to work was the specific release number